set.mm

```diff
@@ -267951,18 +267946,18 @@ the ring of the coefficients is not a division ring (not a field in the
                -> ( x e. I |-> ( ( g ` x ) .x. ( h ` x ) ) ) finSupp .0. ) $=
       ( cv wcel w3a cfv co cmpt cfsupp wbr csupp cfn cof cmap wf 3ad2ant1 simp2
       frlmbasmap syl2anc elmapi syl ffnd simp3 inidm wa eqidd offval oveq1d cvv
-      wfun wss ovexd wceq funmpt a1i funeq mpbird frlmbasfsupp crg cfield isfld
-      cdr ccrg sylib simpld drngring ring0cl ringlz sylan fsuppsssupp fsuppimpd
-      suppofss1d syl22anc eqeltrrd wb mptexd c0g fvexi funisfsupp mp3an2i ) AFU
-      HZLUIZGUHZLUIZUJZBIBUHZXFUKZXKXHUKZEULZUMZOUNUOZXOOUPULZUQUIZXJXFXHEURZUL
-      ZOUPULZXQUQXJXTXOOUPXJBIIXLXMEIXFXHMMXJICXFXJXFCIUSULZUIZICXFUTXJIMUIZXGY
-      CAXGYDXIUGVAZAXGXIVBZLDNICMXFPQSVCVDXFCIVEVFZVGXJICXHXJXHYBUIZICXHUTXJYDX
-      IYHYEAXGXIVHLDNICMXHPQSVCVDXHCIVEVFZVGYEYEIVIXJXKIUIVJZXLVKYJXMVKVLZVMXJX
-      TVNUIZXTVOZXFOUNUOZYAXFOUPULVPZYAUQUIXJXFXHXSVQXJXTXOVRZYMYKYPYMXOVOZYQYP
-      BIXNVSZVTXTXOWAWBVFXJYDXGYNYEYFLDNIMXFOPUBSWCVDXJBICXFXHMEOYEXJDWDUIZOCUI
-      AXGYSXIADWGUIZYSAYTDWHUIZADWEUIYTUUAVJUDDWFWIWJDWKVFVAZCDOQUBWLVFYGYIXJYS
-      XKCUIOXKEULOVRUUBCDEXKOQRUBWMWNWQYLYMVJYNYOVJVJXTOXFXTVNOWOWPWRWSYQXJXOVN
-      UIOVNUIZXPXRWTYRXJBIXNMYEXAUUCXJODXBUBXCVTXOVNVNOXDXEWB $.
+      wfun wss ovexd wceq funmpt funeq mpbiri frlmbasfsupp crg cdr cfield isfld
+      ccrg sylib simpld drngring ring0cl sylan suppofss1d fsuppsssupp fsuppimpd
+      ringlz syl22anc eqeltrrd wb mptexd elexd funisfsupp mp3an2i mpbird ) AFUH
+      ZLUIZGUHZLUIZUJZBIBUHZXEUKZXJXGUKZEULZUMZOUNUOZXNOUPULZUQUIZXIXEXGEURZULZ
+      OUPULZXPUQXIXSXNOUPXIBIIXKXLEIXEXGMMXIICXEXIXECIUSULZUIZICXEUTXIIMUIZXFYB
+      AXFYCXHUGVAZAXFXHVBZLDNICMXEPQSVCVDXECIVEVFZVGXIICXGXIXGYAUIZICXGUTXIYCXH
+      YGYDAXFXHVHLDNICMXGPQSVCVDXGCIVEVFZVGYDYDIVIXIXJIUIVJZXKVKYIXLVKVLZVMXIXS
+      VNUIZXSVOZXEOUNUOZXTXEOUPULVPZXTUQUIXIXEXGXRVQXIXSXNVRZYLYJYOYLXNVOZBIXMV
+      SZXSXNVTWAVFXIYCXFYMYDYELDNIMXEOPUBSWBVDXIBICXEXGMEOYDXIDWCUIZOCUIAXFYRXH
+      ADWDUIZYRAYSDWGUIZADWEUIYSYTVJUDDWFWHWIDWJVFVAZCDOQUBWKVFZYFYHXIYRXJCUIOX
+      JEULOVRUUACDEXJOQRUBWPWLWMYKYLVJYMYNVJVJXSOXEXSVNOWNWOWQWRYPXIXNVNUIOVNUI
+      XOXQWSYQXIBIXMMYDWTXIOCUUBXAXNVNVNOXBXCXD $.
 
```

Comment:
The theorem under focus here is frlmphllem.  Its proof size shrinks from 54 to 52 lines.  Noc change in Syntax Hints, Axioms or Definitions.